kuryr_kubernetes/handlers/retry.py

```diff
@@ -83,6 +83,13 @@ def __call__(self, event, *args, **kwargs):
                 }
                 self._handler(event, *args, retry_info=info, **kwargs)
                 break
+            except (exceptions.LoadBalancerNotReady,
+                    exceptions.PortNotReady):
+                with excutils.save_and_reraise_exception() as ex:
+                    if self._sleep(deadline, attempt, ex.value):
+                        ex.reraise = False
+                    else:
+                        raise
```

Comment:
I guess, some meaningful error message could be logged here depending on exception type - with clear message about either neutron or octavia faults - instead of alarms.

Reply:
It will be logged by [LogExceptions handler](https://github.com/openshift/kuryr-kubernetes/blob/2700a8e58638c076d9df523049a5c3da94dc5366/kuryr_kubernetes/handlers/logging.py#L46) and both exceptions set a meaningful error message in their constructors.

Reply:
Correct I think these will log following:
```
class LoadBalancerNotReady(ResourceNotReady):
     def __init__(self, loadbalancer_id, status):
         super().__init__(
             'Loadbalancer %s is stuck in %s status for several minutes. This '
             'is unexpected and indicates problem with OpenStack Octavia. '
             'Please contact your OpenStack administrator.' % (
                 loadbalancer_id, status))


 class PortNotReady(ResourceNotReady):
     def __init__(self, port_id, status):
         super().__init__(
             'Port %s is stuck in %s status for several minutes. This '
             'is unexpected and indicates problem with OpenStack Neutron. '
             'Please contact your OpenStack administrator.' % (port_id, status))
```